mlfinlab/online_portfolio_selection/momentum/follow_the_leader.py

```diff
@@ -0,0 +1,56 @@
+# pylint: disable=missing-module-docstring
+import numpy as np
+import scipy.optimize as opt
+from mlfinlab.online_portfolio_selection.online_portfolio_selection import OLPS
+
+
+class FollowTheLeader(OLPS):
+    """
+    This class implements the Follow the Leader strategy. It is reproduced with modification from
+    the following paper:
+    Li, B., Hoi, S. C.H., 2012. OnLine Portfolio Selection: A Survey. ACM Comput.
+    Surv. V, N, Article A (December YEAR), 33 pages. <https://arxiv.org/abs/1212.2129>.
+
+    Follow the Leader strategy directly tracks the Best Constant Rebalanced Portfolio until the
+    previous period.
+    """
+
```

Comment:
No `__init__` method?  Is this intended?

Reply:
Yeah, it uses the same parameters as its parent class, so no __init__ method is needed.